server/gaps/helpers.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+
+
+# A python function to create a cold_storage_type_id lookup query based on the 
+# VaccineStorageTemperature one of `2-8°C`, `-20°C`, `-70°C`
+def cold_storage_type_id_lookup(storage_temp):
+    if storage_temp == '2-8°C':
+        return '(SELECT id FROM cold_storage_type WHERE name = \'+5\' or min_temperature = 2 and max_temperature = 8 LIMIT 1)'
+    elif storage_temp == '-20°C':
+        return '(SELECT id FROM cold_storage_type WHERE name = \'-20\' or min_temperature = -20 and max_temperature = -20 LIMIT 1)'
+    elif storage_temp == '-70°C':
+        return '(SELECT id FROM cold_storage_type WHERE name = \'-70\' or min_temperature = -70 and max_temperature = -70 LIMIT 1)'
+    else:
+        return 'null'
+
+
+def create_master_list():
+    print("INSERT INTO master_list (id, name, code, description, is_active) VALUES ('43491ce9-bc89-4ee5-988d-9dbb2856e398', 'gaps_items', 'gaps_items', 'All Items automatically create for Cold Chain Gap Analysis', true) ON CONFLICT DO NOTHING;")
+    print("INSERT INTO master_list_name_join (id, master_list_id, name_link_id) SELECT uuid_in(md5(random()::text || random()::text)::cstring), '43491ce9-bc89-4ee5-988d-9dbb2856e398', id FROM name_link WHERE id NOT IN (select name_link_id from master_list_name_join WHERE master_list_id = '43491ce9-bc89-4ee5-988d-9dbb2856e398');")
```

Comment:
Should make the master list available to all names in the current sync site.
Make sure you sync the site first before running this script.
Seems unlikely that you're not have synced but still...